CHANGELOG.md

```diff
@@ -1,6 +1,22 @@
 # Change Log for Dash for R
 All notable changes to this project will be documented in this file.
 
+
+## [0.3.0] - 2020-02-12
+### Added
+- Support for config-aware relative paths [#172](https://github.com/plotly/dashR/pull/172)
+- Support index customization and index templates [#168](https://github.com/plotly/dashR/pull/168)
+- Application titles may be set using the `app$title()` method, for parity with Dash for Python's `app.title` syntax [#168](https://github.com/plotly/dashR/pull/168)
+
+### Changed
+- Dash for R now requires `dashCoreComponents` v1.8.0
```

Comment:
The table version bump should also be mentioned.

Reply:
fixed in c03c6c1